components/PartnerBenefits.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link";
+
+interface IButtonProps {
+  title: string;
+  linkPath: string;
+}
+
+export default function PartnerBenefits() {
+  function LinkButton({ title, linkPath }: IButtonProps) {
+    return (
+      <Link
+        className="rounded-xl bg-cesium-orange px-6 py-4 text-xl font-bold text-white"
+        href={linkPath}
+      >
+        {title}
+      </Link>
+    );
+  }
+
+  return (
+    <div className="relative mt-6 h-[400px] bg-light-gray px-0 text-center">
+      <h1 className="pt-8 text-2xl font-bold">Vantagens de ser Sócio</h1>
```

Comment:
Change it for a `<h2>` please 🙏🏻, generally the websites will have only one h1 for page, and we should use it in our Hero Section.